src/js/entities-service/entities/organizations.js

```diff
@@ -1,31 +1,24 @@
 import Store from 'backbone.store';
-import Radio from 'backbone.radio';
-import BaseCollection from 'js/base/collection';
 import BaseModel from 'js/base/model';
 
 const TYPE = 'organizations';
 
 const _Model = BaseModel.extend({
-  type: TYPE,
-  urlRoot: '/api/organizations',
-  getGroups() {
-    return Radio.request('entities', 'groups:collection', this.get('_groups'));
+  getStates() {
+    return this.get('states').clone();
   },
   getRoles() {
-    return Radio.request('entities', 'roles:collection', this.get('_roles'));
+    return this.get('roles').clone();
   },
```

Comment:
do we still want to access the states/roles through the organization? That's what's happening here right?